deepchecks/base/display_suite.py

```diff
@@ -61,7 +82,130 @@ def get_display_exists_icon(exists: bool):
     return '<div style="text-align: center">No</div>'
 
 
-def display_suite_result(suite_name: str, results: List[Union[CheckResult, CheckFailure]]):
+def _display_suite_widgets(unique_id: str,
+                           checks_with_conditions: List[CheckResult],
+                           checks_wo_conditions_display: List[CheckResult],
+                           checks_w_condition_display: List[CheckResult],
+                           others_table: List,
+                           light_hr: str):  # pragma: no cover
+    """Display results of suite in as Tab widget."""
+    tab = widgets.Tab()
+    condition_tab = widgets.VBox()
+    _add_widget_classes(condition_tab)
+    checks_wo_tab = widgets.VBox()
+    _add_widget_classes(checks_wo_tab)
+    others_tab = widgets.VBox()
+    tab.children = [condition_tab, checks_wo_tab, others_tab]
+    tab.set_title(0, 'Checks With Conditions')
+    tab.set_title(1, 'Checks Without Conditions')
+    tab.set_title(2, 'Checks Without Output')
+    display_html('<style>.jupyter-widgets.widget-tab > .p-TabBar .p-TabBar-tab {flex: 0 1 auto}</style>',
+                 raw=True)
+
+    if checks_with_conditions:
+        cond_html_h2 = '<h2>Conditions Summary</h2>'
```

Comment:
maybe put all the shared fixed texts outside in a constants

Reply:
done